packages/core/src/AppSwitcherPanel/Action/Action.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+import { useTheme, withStyles } from "@material-ui/core";
+
+import { Info } from "@hitachivantara/uikit-react-icons";
+import { HvTypography, HvAvatar, HvTooltip, HvListItem } from "../..";
+
+import styles from "./styles";
+
+import TitleWithTooltip from "../TitleWithTooltip";
+
+const getColor = (theme, color, defaultColor) => theme.palette[color] || color || defaultColor;
+
+const Action = (props) => {
+  const {
+    id,
+    className,
+    classes,
+    application,
+    onClickCallback = () => {},
+    isSelectedCallback = () => false,
+  } = props;
+
+  const { name, description, disabled, iconElement, iconUrl, url, target } = application;
```

Comment:
maybe we could enhance the api with a `genericProp` to spread extra values to the components below